index.d.ts

```diff
@@ -833,4 +833,7 @@ declare namespace plugins {
  */
 export declare const tracer: Tracer;
 
+export declare const LOG_EXPORTER: 'log-exporter';
```

Comment:
These should not be exported directly from the index, and instead from the `ext` folder similar to other public constants. Unless the idea was to not make them public, in which case the TS definitions are not needed.